Docs 6444 3.9 connector docs (#971)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/anypoint-connectors.adoc

== Connector Support Categories

> MuleSoft provides different support categories. See https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[connector support policies] depending on which connector you use. 

Reviewer: ```suggestion
MuleSoft provides different support categories. See the https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Connector Support Policies] documentation for the connector you use. 
```
